test/operations.py

```diff
@@ -0,0 +1,94 @@
+from http import HTTPStatus
+import requests
+import json
+import os
+
```

Comment:
import logging facility and use info/warn/error levels

Reply:
+1; print statements are usually the wrong answer.  Here's an example of a logger that you can use to get a lot more granularity over what/how things get emitted:

https://github.com/arkime/aws-aio/blob/main/manage_arkime/logging_wrangler.py#L39